src/object/cli_obj.c

```diff
@@ -1895,6 +1937,23 @@ obj_comp_cb(tse_task_t *task, void *data)
 		break;
 	};
 
+	if (obj->cob_time_fetch_leader != NULL &&
+	    ((!obj_is_modification_opc(obj_auxi->opc) &&
+	      task->dt_result == -DER_INPROGRESS) ||
+	     (obj_is_modification_opc(obj_auxi->opc) &&
+	      task->dt_result == 0))) {
+		struct timespec		now;
+		int			idx;
+		int			rc;
+
+		rc = clock_gettime(CLOCK_MONOTONIC_COARSE, &now);
```

Comment:
same, seems can reuse d_time2s

Reply:
Niu has a function get_current_age(), which does the same thing, you could move it to the common library and reuse it.

Reply:
OK. I will update the patch.